sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -86,6 +86,9 @@ case class RowDataSourceScanExec(
 
   def output: Seq[Attribute] = requiredColumnsIndex.map(fullOutput)
 
+  override val nodeName: String =
```

Comment:
`DataSourceScanExec.nodeName` is defined as `s"Scan $relation ${tableIdentifier.map(_.unquotedString).getOrElse("")}"`, do we really need to overwrite it here?

Reply:
My intent was to be able to distinguish between RowDataSourceScan and FileSourceScan. Removed those overrides.